.github/workflows/ci.yml

```diff
@@ -164,22 +164,15 @@ jobs:
     if: github.event_name != 'release' && github.event.action != 'published'
     strategy:
       matrix:
-        package: ${{ fromJSON(needs.get-workspace-members.outputs.members) }}
         include:
+          - command: clippy
+            args: --all-targets
           - command: fmt
             args: --all --verbose -- --check
-          - command: clippy
-            args: --all-features --all-targets
-          - command: check
-            args: --all-features --locked --workspace --all-targets
-          - command: build
-            args: --locked --workspace --all-features --all-targets
-          - command: test
-            args: --locked --workspace
+          # Excluding fuel-indexer-tests package as the SDK macros cause
```

Comment:
Same comment here regarding putting this in a matrix but not running it on a per-package basis